docker_templates/templates/docker_images/create_base_image.Dockerfile.em

```diff
@@ -36,7 +36,7 @@ ENV TZ @timezone
 RUN rosdep init
 
 # install requested metapackage
-RUN apt-get update && apt-get install -q -y @(' '.join(packages))@
+RUN apt-get update && apt-get install -q -y --no-install-recommends @(' '.join(packages))@
```

Comment:
Was this template always missing the  `&& rm -rf /var/lib/apt/lists/*`? Where is this template used again?

Reply:
It's not used anywhere in `docker_images` or `docker_templates` at the moment. But I'm not sure what it was used for before